lms/djangoapps/bookmarks/urls.py

```diff
@@ -0,0 +1,27 @@
+"""
+Unit Bookmark API URLs
+"""
+
+from django.conf import settings
+from django.conf.urls import patterns, url
+
+from .views import BookmarksView, BookmarksDetailView
+
+USERNAME_PATTERN = r'(?P<username>[\w.@+-]+)'
```

Comment:
Can you move this to `settings.py`. I see it duplicated in a number of places in the codebase.